test/api.ts

```diff
@@ -355,4 +355,27 @@ suite('api v1', function () {
             // ])
         }
     })
+
+    test('history get_actions', async function () {
+        const res = await eos.v1.history.get_actions('teamgreymass', 1, 1);
+        assert.equal(res.actions.length, 1)
+        assert.equal(res.last_irreversible_block, 111);
+    })
+
+    test('history get_transaction', async function () {
+        const res = await eos.v1.history.get_transaction(Checksum256.from('03ef96a276a252b66595d91006ad0ff38ed999816f078bc5d87f88368a9354e7'))
+        assert.equal(res.id, "03ef96a276a252b66595d91006ad0ff38ed999816f078bc5d87f88368a9354e7")
+        assert.equal(res.block_num, 111)
+    })
+
+    test('history get_key_accounts', async function () {
+        const res = await eos.v1.history.get_key_accounts('PUB_K1_6RrvujLQN1x5Tacbep1KAk8zzKpSThAQXBCKYFfGUYeACcSRFs')
```

Comment:
Should we do this with a bunch of different key formats to ensure they all work? 

For example, `EOS...` and `FIO...` prefix keys.

Reply:
We could! Can the FIO prefix key be used on EOS? Or are you saying that we should hit the apis from different chains?

Reply:
Keys are normalized to PUB_ when converting to JSON and we have tests covering those conversions already so I don't think that will be necessary